Description: fix format string use
 C4Aul's Warn uses the format attribute to check for format string
 correctness. Passing a dynamic string buffer from FormatString to it
 will break the build with -Werror=format-string. Given that Warn already
 does the formatting itself, just drop the use of FormatString.
Author: Philipp Kern <pkern@debian.org>
Origin: vendor
Applied-Upstream: http://hg.openclonk.org/openclonk/rev/95d5a2fecb11

--- openclonk-5.3.1.orig/src/script/C4AulParse.cpp
+++ openclonk-5.3.1/src/script/C4AulParse.cpp
@@ -1264,7 +1264,7 @@ void C4AulParse::Parse_Script()
 			case ATT_DIR:
 			{
 				if (found_code)
-					Warn(FormatString("Found %s after declarations", Idtf).getData());
+					Warn("Found %s after declarations", Idtf);
 				// check for include statement
 				if (SEqual(Idtf, C4AUL_Include))
 				{
@@ -1477,7 +1477,7 @@ void C4AulParse::Parse_Function()
 		{
 			if (Type == PREPARSER) Fn->AddPar(Idtf);
 			if (Config.Developer.ExtraWarnings)
-				Warn(FormatString("'%s' used as parameter name", Idtf).getData());
+				Warn("'%s' used as parameter name", Idtf);
 		}
 		else if (TokenType != ATT_IDTF)
 		{
